docs/user-guide/03-convenience-features.md

```diff
@@ -361,7 +361,7 @@ float4 myPackedVector = reinterpret<float4>(myVal);
 
 `reinterpret` can pack any type into any other type as long as the target type is no smaller than the source type.
 
-## Pointers
+## Pointers (limited)
 
 Slang supports pointers when generating code for SPIRV, C++ and CUDA targets. The syntax for pointers is similar to C, with the exception that operator `.` can also be used to dereference a member from a pointer. For example:
 ```csharp
```

Comment:
The example code below doesn't work.
The variable name `pNext` is declared twice.